.github/workflows/pages.yml

```diff
@@ -1,5 +1,5 @@
 name: Pages
-on: [push]
+on: [push, pull_request]
```

Comment:
why would we run this on a pull request if it is specified below that it should only run on main?

Reply:
we only deploy the docs for commits on main. But for development purposes we build them also for pull requests and upload them as an artifact